src/crimson/os/seastore/cache.h

```diff
@@ -280,33 +291,33 @@ class Cache {
     auto cached = query_cache(offset, p_src_ext);
     if (!cached) {
       auto ret = CachedExtent::make_cached_extent_ref<T>(
-        alloc_cache_buf(length));
+        length);
```

Comment:
Looks this is where (also line 313 below) the new constructor `CachedExtent(extent_len_t length)` is used. Why the original `CachedExtent::make_cached_extent_ref<T>(alloc_cache_buf(length))` cannot work?